parser/limits.in

```diff
@@ -1,15 +1,13 @@
-key=value
+; minimum supported section name length of 1024 characters
+[{test3,aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa}]
```

Comment:
There are only 1014 `a`s in this string, and the total length inside the brackets is 1022.  If I read https://github.com/editorconfig/specification/pull/21/files#diff-db4b047b30ffef7855b12b527860ac2cR134 correctly, there should be 1024 characters inside the brackets here, correct?

Reply:
Good catch. The old limit was tied to the length of the glob without brackets - at least that's how it was implemented in `ec4j`. I wonder if we should make this more explicit in the spec?

Reply:
> I wonder if we should make this more explicit in the spec?

Reading again, I think it is clear enough.

Reply:
Good catch indeed! I based the tests on the existing ones, but confused test3 and test4. 🤦 

I fixed this and also added note in the comment in the `limits.in`
```
; minimum supported section name length of 1024 characters (excluding [] brackets)
```

---

> Reading again, I think it is clear enough.

Agree. It directly follows from
> Section Name: the string between the beginning ``[`` and the ending ``]``.

Reply:
@cxw42 can you take a look at this, please? I believe it should be all good now.

I also run the '[limitless branch](https://github.com/ec4j/ec4j/pull/111)' of ec4j against this branch and everything passed.  (thanks @ppalaga )